### ISSUE TITLE
Inconsistency timestep types in SQL ReportDataDictionary table and actual outputs

### ISSUE THREAD

**Issue by hongyuanjia** [NREL/EnergyPlus#8227]
Issue overview
--------------

### Details
 - Platform: Windows 10
 - Version of EnergyPlus: 9.1

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added ([5Zone_Transformer_Detailed.idf.txt](https://github.com/NREL/EnergyPlus/files/5118015/5Zone_Transformer_Detailed.idf.txt))

The `TimestepType` column in SQL `ReportDataDictionary` seems to not give the correct output timestep. In my case, I added one output variable and two output meters, with `Detailed` reporting frequency, as shown below:

```
!-   ===========  ALL OBJECTS IN CLASS: OUTPUT:VARIABLE ===========

Output:Variable,
    *,                       !- Key Value
    Zone Air Heat Balance Internal Convective Heat Gain Rate,  !- Variable Name
    Detailed;                !- Reporting Frequency


!-   ===========  ALL OBJECTS IN CLASS: OUTPUT:METER ===========

Output:Meter,
    Electricity:Facility,    !- Key Name
    Detailed;                !- Reporting Frequency


!-   ===========  ALL OBJECTS IN CLASS: OUTPUT:METER:CUMULATIVE ===========

Output:Meter:Cumulative,
    Electricity:Facility,    !- Key Name
    Detailed;                !- Reporting Frequency
```

The `ReportDataDictionary` table in SQL output indicates that these two meters should be output in `HVAC System` timestep, and others are in `Zone` timestep. Please see below:

```
#    TimestepType   ReportingFrequency    KeyValue
#          <char>               <char>      <char>
# 1:  HVAC System HVAC System Timestep            
# 2:  HVAC System HVAC System Timestep Cumulative 
# 3:         Zone HVAC System Timestep    PLENUM-1
# 4:         Zone HVAC System Timestep    SPACE1-1
# 5:         Zone HVAC System Timestep    SPACE2-1
# 6:         Zone HVAC System Timestep    SPACE3-1
# 7:         Zone HVAC System Timestep    SPACE4-1
# 8:         Zone HVAC System Timestep    SPACE5-1
#                                                        Name
#                                                      <char>
# 1:                                     Electricity:Facility
# 2:                                     Electricity:Facility
# 3: Zone Air Heat Balance Internal Convective Heat Gain Rate
# 4: Zone Air Heat Balance Internal Convective Heat Gain Rate
# 5: Zone Air Heat Balance Internal Convective Heat Gain Rate
# 6: Zone Air Heat Balance Internal Convective Heat Gain Rate
# 7: Zone Air Heat Balance Internal Convective Heat Gain Rate
# 8: Zone Air Heat Balance Internal Convective Heat Gain Rate
```

While in the actual CSV output, the timestep is opposite as stated in the `ReportDataDictionary`: the meters are shown in `Zone` timestep while others are shown in `HVAC System` timestep. Below is row 30 to 40 in the CSV file:

|Date/Time       | PLENUM-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| SPACE1-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| SPACE2-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| SPACE3-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| SPACE4-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| SPACE5-1:Zone Air Heat Balance Internal Convective Heat Gain Rate [W](Each Call)| Electricity:Facility [J](Each Call)| Cumulative Electricity:Facility [J](Each Call)|
|:---------------|--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------:|-----------------------------------:|----------------------------------------------:|
|01/01  07:30:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                             1606484|                                       48194509|
|01/01  07:37:30 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|
|01/01  07:45:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                             1606484|                                       49800993|
|01/01  07:52:30 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|
|01/01  08:00:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                             1606484|                                       51407476|
|01/01  08:07:30 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|
|01/01  08:15:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                             1606484|                                       53013960|
|01/01  08:20:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|
|01/01  08:25:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|
|01/01  08:30:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                             1606484|                                       54620444|
|01/01  08:35:00 |                                                                                0|                                                                          148.632|                                                                           64.182|                                                                          148.632|                                                                           64.182|                                                                          278.122|                                  NA|                                             NA|

Is this the intended behavior? Thanks

**Comment by mjwitte** [comment]
I believe this was fixed for v9.2 in https://github.com/NREL/EnergyPlus/pull/7415.

**Comment by hongyuanjia** [comment]
@mjwitte Thanks. Yes, this has been fixed in v9.2.
